src/EnergyPlus/Autosizing/CoolingCapacitySizing.cc

```diff
@@ -140,8 +140,7 @@ Real64 CoolingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
                                     CoilInTemp = this->finalZoneSizing(this->curZoneEqNum).DesCoolCoilInTemp;
                                     CoilInHumRat = this->finalZoneSizing(this->curZoneEqNum).DesCoolCoilInHumRat;
                                 } else {
-                                    CoilInTemp = this->finalZoneSizing(this->curZoneEqNum)
-                                                     .ZoneRetTempAtCoolPeak; // Question whether zone equipment should use return temp for sizing
+                                    CoilInTemp = this->finalZoneSizing(this->curZoneEqNum).ZoneTempAtCoolPeak;
```

Comment:
This has been wrong and ripe for change for some time. Zone equipment draws directly from the zone via the zone exhaust node, hence, zone temp.